README.md

```diff
@@ -8,6 +8,7 @@ Teams spend significant time building parts of the design system (colors, variab
 [![CircleCI](https://circleci.com/gh/Microsoft/fast-dna/tree/master.svg?style=shield&circle-token=d159a8b24ccb8046e07138c98717c32cb92589d6)](https://circleci.com/gh/Microsoft/fast-dna/tree/master)
 [![Maintainability](https://api.codeclimate.com/v1/badges/8a74621e634a6e9b9561/maintainability)](https://codeclimate.com/github/Microsoft/fast-dna/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/8a74621e634a6e9b9561/test_coverage)](https://codeclimate.com/github/Microsoft/fast-dna/test_coverage)
+[![Build Status](https://saucelabs.com/buildstatus/FAST-DNA)](https://saucelabs.com/beta/builds/997f70860d604d83b3f5c69129216d02)
```

Comment:
This is interesting. I'm all for badges if they convey valuable information. When I view this, it shows:

"SauceLabs: complete"

Is this valuable? What does complete mean in the sense of the repository?

Reply:
@nicholasrice or @janechu for thoughts

Reply:
It is required for Open Sauce (open source) projects using Sauce Labs.  But, I can see your point since we're no longer plugging this directly into our pipeline.  I guess the good thing is it will still show status for the past person that ran it from their dev system.

Sometimes these don't always work until they are actually committed in readme.  If it doesn't I can fix it with another PR.

Reply:
gotcha